static/app/views/performance/transactionSummary/transactionOverview/charts.tsx

```diff
@@ -199,8 +174,7 @@ function TransactionSummaryCharts({
             end={eventView.end}
             statsPeriod={eventView.statsPeriod}
             currentFilter={currentFilter}
-            queryExtras={queryExtras}
```

Comment:
yeah because for now, histograms are powered by indexed events since they're more accurate. it's a short term behavior until metrics histograms are sorted out